e2e-polybft/framework/test-server.go

```diff
@@ -198,14 +198,14 @@ func (t *TestServer) Stop() {
 }
 
 // RootchainFund funds given validator account on the rootchain
-func (t *TestServer) RootchainFund(rootNativeERC20Addr types.Address, amount *big.Int) error {
+func (t *TestServer) RootchainFund(stakeToken types.Address, amount *big.Int) error {
 	args := []string{
 		"rootchain",
 		"fund",
 		"--addresses", t.address.String(),
 		"--amounts", amount.String(),
 		"--json-rpc", t.BridgeJSONRPCAddr(),
-		"--native-root-token", rootNativeERC20Addr.String(),
+		"--stake-token", stakeToken.String(),
```

Comment:
Same here, can we get it from genesis?

Reply:
As per comment: https://github.com/0xPolygon/polygon-edge/pull/1520#discussion_r1200858670